var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import glob
```

Comment:
@davydden Apparently flake has become more strict: it is complaining about this unused import.